options/locale/locale_en-US.ini

```diff
@@ -2331,6 +2331,7 @@ mirror_sync_create = synced new reference <a href="%s/src/%s">%[2]s</a> to <a hr
 mirror_sync_delete = synced and deleted reference <code>%[2]s</code> at <a href="%[1]s">%[3]s</a> from mirror
 approve_pull_request = `approved <a href="%s/pulls/%s">%s#%[2]s</a>`
 reject_pull_request = `suggested changes for <a href="%s/pulls/%s">%s#%[2]s</a>`
+publish_release  = `published release <a href="%s/releases/tag/%s"> "%[4]s" </a> to <a href="%[1]s">%[3]s</a>`
```

Comment:
Why quotes needed here?

```suggestion
publish_release  = `published release <a href="%s/releases/tag/%s">%[4]s</a> to <a href="%[1]s">%[3]s</a>`
```

Reply:
Because some projects not only use single tag names as release titles, like "XXX vx.x.x released", so I think quotes are necessary.

Reply:
Maybe use tag name is better.
![image](https://user-images.githubusercontent.com/25342410/88743289-e1088700-d176-11ea-9bec-29b48f2dce97.png)